store/LoginRegister.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Login/Register</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="/main.css" />
+        <link rel="stylesheet" href="/styles/LoginRegister.css" />
+    </head>
+    <body>
+     
+        <nav>
+            <div class="logo">Scents</div>
+            <ul class="menu">
+              <li><a href="http://127.0.0.1:5500/">Home</a></li>
+              <li><a href="/index.html/#banner-section">About Us</a></li>
+              <li><a href="/store/productdetails.html">Products</a></li>
+              <svg class="cart__icon" viewBox="0 0 24 24">
+                <path fill="currentColor" d="M12,13A5,5 0 0,1 7,8H9A3,3 0 0,0 12,11A3,3 0 0,0 15,8H17A5,5 0 0,1 12,13M12,3A3,3 0 0,1 15,6H9A3,3 0 0,1 12,3M19,6H17A5,5 0 0,0 12,1A5,5 0 0,0 7,6H5C3.89,6 3,6.89 3,8V20A2,2 0 0,0 5,22H19A2,2 0 0,0 21,20V8C21,6.89 20.1,6 19,6Z" />
+            </svg>
+            </ul>
+      
+          
+      
+            <button class="btn"> <a href="/store/LoginRegister.html"> Login/Register</a></button>
+          </nav>
+        
+          <header id="banner">
+            <div class="banner">
+              <h1 class="banner__title">
+                Elevate Your Senses with Exquisite Fragrances
+              </h1>
+            </div>
+          </header>
+
+
+
+          <div class="login-register-page">
+            <div class="form-container login-container">
+              <form>
+                <h1>Login</h1>
```

Comment:
try to follow the heading hierarchy, so once you already have an h1 element in the page, try to fix this to h2 or h3

Reply:
yeah sure, I have not completed it as i still need to push some changes and clean up. Had to submit because of the time, but you already graded it haha